.classpath

```diff
@@ -6,14 +6,7 @@
 			<attribute name="gradle_used_by_scope" value="main,test"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" output="bin/test" path="src/test/java">
-		<attributes>
-			<attribute name="gradle_scope" value="test"/>
-			<attribute name="gradle_used_by_scope" value="test"/>
```

Comment:
Why did we delete this

Reply:
Not a human-controlled file, was auto done by either vscode or gradle.